### PR TITLE
Dashboard: remove html5 icon, remove z-index for overlay

### PR DESCRIPTION
**Pull request by resetnow**
Signed-off-by: Vlad Ivanov <vlad@ivanov.email>

### CODE REVIEW

**Comment by kaikreuzer** [comment]
Hm, you are not removing the overlay here, but you are removing the mechanism that allows to add an overlay.
The Tiles still can define overlays, but the dashboard now wouldn't now how to handle them.
I think in general, we can keep the mechanism in place, wdyt? If not, we would also have to adapt the DashboardTile class.

**Comment by resetnow** [comment]
I left the overlay element itself because it adds the gradient. As for getOverlay method, it may be useful if the tile has a logo, for example, or if the tile has a specific purpose (configuration-only interface may have a gear icon).

File: bundles/org.openhab.ui.dashboard/src/main/java/org/openhab/ui/dashboard/internal/DashboardServlet.java [hunk 1]
```diff
@@ -75,7 +75,6 @@ private void serveDashboard(HttpServletRequest req, HttpServletResponse resp) th
         for (DashboardTile tile : tiles) {
             String entry = entryTemplate.replace("<!--name-->", tile.getName());
             entry = entry.replace("<!--url-->", tile.getUrl());
-            entry = entry.replace("<!--overlay-->", tile.getOverlay());
```
Comment (kaikreuzer) [inline bundles/org.openhab.ui.dashboard/src/main/java/org/openhab/ui/dashboard/internal/DashboardServlet.java]: So why do you remove this line? This means that a tile overlay won't have any effect anymore.
Imho, we should simply allow getOverlay() to return null and handle that accordingly.

Comment (resetnow) [inline bundles/org.openhab.ui.dashboard/src/main/java/org/openhab/ui/dashboard/internal/DashboardServlet.java]: Makes sense, I'll update the PR.

**Comment by resetnow** [comment]
There are some trailing spaces removed, I can remove this out of commit if needed.

**Pull request merged**